src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResourceIntTest.kt

```diff
@@ -180,4 +192,96 @@ class UserResourceIntTest : IntegrationTestBase() {
       }
     }
   }
+
+  @Nested
+  @DisplayName("POST /users")
+  inner class MaintainUsers {
+
```

Comment:
maybe add a test to check it has the correct role?

Reply:
Yes will do